DXRModules/DeusEx/Classes/DXREnemiesBase.uc

```diff
@@ -238,16 +238,20 @@ function ScriptedPawn CloneScriptedPawn(ScriptedPawn p, optional class<ScriptedP
     }
     if( newclass == None ) newclass = p.class;
     newtag = StringToName(p.Tag $ "_clone");
-    radius = p.CollisionRadius + newclass.default.CollisionRadius;
+    radius = (p.CollisionRadius + newclass.default.CollisionRadius) * 3.0;// combined radius, and * 3 for some personal space
+    num_enemies = float(enemy_multiplier) * float(dxr.flags.settings.enemiesrandomized+100) / 100.0;
+    radius *= Sqrt(num_enemies+1.0);
     for(i=0; i<10; i++) {
-        loc_offset.X = 1 + rngf() * 3 * Sqrt(float(enemy_multiplier+1));
-        loc_offset.Y = 1 + rngf() * 3 * Sqrt(float(enemy_multiplier+1));
-        if( chance_single(50) ) loc_offset *= -1;
+        // rngfn_min_dist to add a minimum distance from the 0, but also allow negative numbers
+        loc_offset.X = rngfn_min_dist(0.5);
```

Comment:
we used to use a +1 instead of +0.5, but we've moved where the *3 happens so really this is a +1.5, we have more spread now than before, and on top of that we now factor in the enemiesrandomized flag for how wide the radius should be